packages/w3up-client/src/service.js

```diff
@@ -30,11 +30,32 @@ export const filecoinServiceConnection = client.connect({
   channel: HTTP.open({ url: accessServiceURL, method: 'POST' }),
 })
 
+export const gatewayServiceURL = new URL('https://storacha.link')
+export const gatewayServicePrincipal = DID.parse('did:web:storacha.link')
```

Comment:
We will probably need to keep the `did:web:w3s.link` because the Gateway is configured to verify that DID.

Reply:
👍 will change back